lib/desired-caps.js

```diff
@@ -0,0 +1,102 @@
+import log from './logger';
+import validator from 'validate.js';
+
+validator.validators.isString = function(value) {
```

Comment:
Yeah but it's like:
```
require('validate.js').validators.isString = function(value)
```
So our custom functions are hanging from the npm cache. Cleaner way in node would be:
```
validator = require('validate.js')()
validator.validators.isString = function(value)
```